style.css

```diff
@@ -379,6 +379,10 @@ main h2, main h3, main h4, main h5, main h6 {
     border-top: 2px solid #f0a202;
 }
 
+.noncomputable {
+    border-left-style: dashed;  /* TODO: purple instead to match the noncomputable keyword? */
+}
+
```

Comment:
Reid expressed a slight dislike of this (or any) special styling for `noncomputable`.

Should we just remove this and merge, and leave someone else to bikeshed styling in a follow-up?

Reply:
```suggestion
```